test/common/router/header_parser_corpus/address_0

```diff
@@ -1,6 +1,6 @@
 headers_to_add {
   header {
     key: "x-request-start"
-    value: "%DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT%"
```

Comment:
how come you changed the existing corpus files instead of just adding a new one with the new option?

Reply:
yeah, I could add a new one without modifying existing ones, but I wanted to follow the order of:
address_0: DOWNSTREAM_REMOTE_ADDRESS  <=== added by this PR
address_1: DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT
address_2: DOWNSTREAM_LOCAL_ADDRESS
address_3: DOWNSTREAM_LOCAL_ADDRESS_WITHOUT_PORT

That's why I had to modify all existing address_* files. Note that I am not very familiar with fuzz test, and I assume those files are used as test inputs and are manually generated.

Reply:
I think this is probably fine. @htuch any reason we shouldn't be changing existing corpus entries like this?

Reply:
Corpus entries often act as regressions. Generally I'd recommend against modifying existing entries, but if you do, please at least retain existing coverage and just do it additively.

Reply:
okay, reverted the changes